ej2-react/gantt/data-binding.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Data binding in React Gantt component | Syncfusion
-description: Learn here all about Data binding in Syncfusion React Gantt component of Syncfusion Essential JS 2 and more.
+title: Data binding in React Gantt Component | Syncfusion
+description: Learn here all about Data binding in React Gantt component of Syncfusion Essential JS 2,its elements, and more.
```

Comment:
Update like this - Learn here all about data binding support in React Gantt component of Syncfusion Essential JS2, its elements, and more.